pkg/controller/compute/kubernetes/aws_handler.go

```diff
@@ -66,10 +65,11 @@ func (r *AWSClusterHandler) provision(class *corev1alpha1.ResourceClass, instanc
 	return cluster, err
 }
 
-// bind updates resource state binding phase
+// SetBindStatus updates resource state binding phase
 // - state = true: bound
 // - state = false: unbound
-func (r AWSClusterHandler) setBindStatus(name types.NamespacedName, c client.Client, state bool) error {
+// TODO: this setBindStatus function could be refactored to 1 common implementation for all providers
+func (r AWSClusterHandler) SetBindStatus(name types.NamespacedName, c client.Client, state bool) error {
```

Comment:
I feel that we could get away without the explanatory comment around what the state boolean means for the binding state by simply renaming it `bound`. i.e.:

```go
func (r AWSClusterHandler) SetBindStatus(name types.NamespacedName, c client.Client, bound bool) error
```

Reply:
That's true and an easy (non-functional) change to make, i'll incorporate that too.